sentinelsat/sentinel.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, user, password, api_url='https://scihub.copernicus.eu/apihub/
         self._last_query = None
         self._last_response = None
 
-    def query(self, area=None, date=None, raw=None, area_relation='Intersects',
```

Comment:
No, please don't change the query parameter name `date`. Rather do `import datetime` and `datetime.date`, which we should be doing anyways, IMHO.

Reply:
Agreed, I'll rewrite it that way then. But there are several usages of datetime.datetime.<something>. It seems that these imports won't be really nice anyway.

Reply:
I prefer `datetime.datetime`, because just `datetime` is so ambiguous. Weird that they did not call it `datetime.Datetime`, since it is a class, isn't it?

Reply:
Absolutely, I suppose somewhere deep in the python community we can find heated arguments about this naming:)